services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -117,6 +120,7 @@ export const AddEditReportModal = ({
         fieldData: {
           ...dataToWrite.fieldData,
           stateName: States[activeState as keyof typeof States],
+          submissionCount: reportType === "MLR" ? 0 : undefined,
```

Comment:
will NAAAR need submission counts? I would prefer the front end and back end both be program agnostic whenever possible, leveraging configuration flags rather than hardcoding report type toggles.

Reply:
Talked to DG about this—he said we're a long way off from knowing.